src/components/topbar.tsx

```diff
@@ -134,7 +140,20 @@ const Topbar = ({
                                 </NavDropdown.Item>
                             </NavDropdown>
                         )}
-                        {!username && <TwitchLoginButton url="/api" />}
+                        {sessionError && (
+                            <div className="ms-2">
+                                <button
+                                    className="btn btn-primary"
+                                    type="button"
+                                    onClick={handleResetSession}
+                                >
+                                    Reset session
+                                </button>
+                            </div>
+                        )}
+                        {!username && !sessionError && (
```

Comment:
Seeing L143 with sessionError, we could swap this to !sessionError && !username just for readability

```suggestion
                        {!sessionError && !username && (
```